core-bundle/src/Resources/contao/languages/en/exception.xlf

```diff
@@ -47,6 +47,30 @@
       <trans-unit id="XPT.maintenance">
         <source>The website is currently not available. Please come back later.</source>
       </trans-unit>
+      <trans-unit id="XPT.missingRouteParameters.title">
+        <source>Unable to generate front end URL</source>
+      </trans-unit>
+      <trans-unit id="XPT.missingRouteParameters.matter">
+        <source>Contao cannot generate the URL for a front end page, because it requires parameters that are unknown. The page cannot be visited without additional information, like a news alias in the URL.</source>
```

Comment:
```suggestion
        <source>The page could not be generated, because the route has mandatory parameters that are not given. Additional information (e.g. a news alias) must be provided in the URL.</source>
```

Reply:
Unfortunately, that is not true. The exception happens when the URL is tried to be generated, not when the page is rendered.

Reply:
I have adjusted my suggestion.